toothpick-runtime/src/main/java/toothpick/Toothpick.java

```diff
@@ -39,12 +39,31 @@
   // its transformation
   private static final ConcurrentHashMap<Object, Scope> MAP_KEY_TO_SCOPE =
       new ConcurrentHashMap<>();
+  private static final ConcurrentHashMap<Object, Scope> ROOT_SCOPES = new ConcurrentHashMap<>();
```

Comment:
Don't we need only a set ?

Reply:
concurrenthashset does not exist

Reply:
Ok, thx. Then add a comment in the code about the usage of a map in this case.